autolens/imaging/plot/fit_imaging_plotters.py

```diff
@@ -196,7 +196,7 @@ def figures_2d_of_planes(
 
             if model_image:
 
-                if (not self.tracer.has(cls=aa.pix.Pixelization)) or plane_index == 0:
+                if (not self.tracer.has(cls=aa.Pixelization)) or plane_index == 0:

```

Comment:
These brackets are redundant although I guess you could argue they make it clearer